models/intel/horizontal-text-detection-0001/description/horizontal-text-detection-0001.md

```diff
@@ -0,0 +1,50 @@
+# text-detection-0003
+
+## Use Case and High-Level Description
+
+Text detector based on [FCOS](https://arxiv.org/abs/1904.01355) architecture with [MobileNetV2-like](https://arxiv.org/abs/1801.04381) as a backbone for indoor/outdoor scenes with more or less horizontal text.
+
+The key benefit of this model compared to the [base model](../../text-detection-0003/description/text-detection-0003.md) is its smaller size and faster performance.
+## Example
+
+![](./horizontal-text-detection-0001.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| F-measure (harmonic mean of precision and recall on ICDAR2013)| 88.45%                 |
+| GFlops                                                        | 7.78                  |
+| MParams                                                       | 2.26                   |
+| Source framework                                              | PyTorch              |
+
+
+## Performance
+
+## Inputs
+
+1. name: "input" , shape: [1x3x704x704] - An input image in the format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - RGB.
```

Comment:
all models in OMZ have BGR color order (even if original model has RGB input, it will be distributed as IR converted with `--reverse_input_channels`).
Do you have some specific reasons to stay input in RGB (e.g. MO can not convert it with this option or any other issues)?

Reply:
No, there are no specific reasons to keep it. But if I change order to BGR, should I fix any configs? And if the answer is yes, which files must be modified?